checkmk_extensions/agent/robotmk_collector.ps1

```diff
@@ -6,23 +6,52 @@ function Main {
         [parameter(Mandatory=$true, Position=0)]
         [string]$ConfigPath
     )
+    Write-Output "<<<robotmk_v2:sep(10)>>>"
 
-    $resultsDirectory = GetResultsDirectory($ConfigPath)
-    $files = Get-ChildItem -File -Recurse $resultsDirectory
+    try {
+        $configFileContent = Get-Content -Raw -Path $ConfigPath
+    }
+    catch {
+        Write-Output (SerializeConfigReadingError($Error[0]))
+        Throw $Error[0]
```

Comment:
OTI: I guess this changes the exit code, but the windows agent still produces the output? Do we really want exit code 1.

Reply:
Yes, the Windows agent still produces output if the plugin exits with 1, just tried it out. We already had exit code 1 before this change. `GetResultsDirectory` can throw an exception (for example if `results_directory` is missing or if the configuration isn't valid JSON).